internal/policy/root_test.go

```diff
@@ -0,0 +1,104 @@
+package policy
+
+import (
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/adityasaky/gittuf/internal/tuf"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestInitializeRootMetadata(t *testing.T) {
+	keyBytes, err := os.ReadFile(filepath.Join("test-data", RootPublicKeysTreeEntryName, "437cdafde81f715cf81e75920d7d4a9ce4cab83aac5a8a5984c3902da6bf2ab7"))
```

Comment:
Does the file need to be named `437cdafde81f715cf81e75920d7d4a9ce4cab83aac5a8a5984c3902da6bf2ab7` for things to work? If so, how would someone regenerate this?

Reply:
It's stored with the key ID as the file name. The idea is that when you use `gittuf trust...` and you're adding your key, it also stores the key in the appropriate form. Within `test-data`, I've structured the files to match the `refs/gittuf/policy` tree.